src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -219,6 +222,45 @@ void Engine::ApplyPerformanceHints(std::map<std::string, std::string> &config, c
         return std::pair<std::string, Engine::StreamCfg>(std::to_string(streams_info.num_streams), streams_info);
     };
 
+    auto getNumStreams = [&](int nstreams) {
```

Comment:
As long as we have `cpu_streams_caclutalition` file now can we move all realted logic from `plugin.cpp` there?

Reply:
Ok, done.